docs/overview/ocean_biogeo_cyc.rst

```diff
@@ -0,0 +1,21 @@
+Ocean biogeochemical cycles
+---------------------------
+
+
+MITgcm is being used to study global biogeochemical cycles in the
+ocean. For example one can study the effects of interannual changes in
+meteorological forcing and upper ocean circulation on the fluxes of
+carbon dioxide and oxygen between the ocean and atmosphere. :numref:`biogeo_figure`
+shows the annual air-sea flux of oxygen and its
+relation to density outcrops in the southern oceans from a single year
+of a global, interannually varying simulation. The simulation is run
+at 1°x1° resolution telescoping to ⅓°x⅓° in the tropics (not shown).
```

Comment:
These fractions (and the others in "ocean_gyres.rst")  didn't work well for me when I tried to build the pdf version of the manual. They turn out just fine in the html though.

Perhaps it would be better to explicitly render them using the :math:`` environment?